src/features/nft/nftModals/bidHistoryModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import cn from 'classnames'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import Scrollbar from '../../../common/components/Scrollbar'
+import Bid from '../../../common/components/Bid'
+
+type THistoryItem = {
+  id: string | number
+  avatar: string
+  name: string
+  bid: string | number
+  date: string
+}
+
+export type TBidHistoryModal = {
+  history: Array<THistoryItem>
```

Comment:
Can we use ` history: THistoryItem[]` instead of `history: Array<THistoryItem>`?